test/utils/utils.go

```diff
@@ -320,11 +320,13 @@ func CreateTempDirInTempDir() (string, error) {
 }
 
 // SystemExec is used to exec a system command to check its exit code or output
-func SystemExec(command string, args []string) *PodmanSession {
+func SystemExec(waitComplete bool, command string, args []string) *PodmanSession {
```

Comment:
So, I see no instances of `SystemExec(false`. Is it remotely conceivable that anyone will ever need this? If not, why not save a lot of work and confusion (and future confusion from anyone who uses `SystemExec()` in the future and forgets to add `true`); why not just unconditionally do the `Wait`?

Reply:
I agree, I don't think anyone would ever make the assumption to not want to wait for the command to be finished.

Reply:
Yes, but then how could I do things the hard way?

Reply:
Okay, removed all'em and fixed another test bug.  Thanks :smile: